modules/installation-aws-add-iam-roles.adoc

```diff
@@ -0,0 +1,41 @@
+// Module included in the following assemblies:
+//
+// * installing/installing_aws/installing-aws-account.adoc
+
+:_content-type: PROCEDURE
+[id="specify-an-existing-iam-role_{context}"]
+= Specifying an existing IAM role
+
+Instead of allowing the installation program to create IAM instance profiles with the default permissions, you can use the `install-config.yaml` file to specify an existing IAM role for control plane and compute instances.
+
+.Prerequisites
+
+* You have an existing `install-config.yaml` file.
+
+.Procedure
+
+. Update `compute.platform.aws.iamRole` with an existing role for the control plane machines.
++
+.Sample `install-config.yaml` file with an IAM role for compute instances
+[source,yaml]
+----
+compute:
+- hyperthreading: Enabled
+  name: worker
+  platform:
+    aws:
+      iamRole: ExampleRole
+----
+. Update `controlPlane.platform.aws.iamRole` with an existing role for the compute machines.
++
+.Sample `install-config.yaml` file with an IAM role for control plane instances
+[source,yaml]
+----
+controlPlane:
+  hyperthreading: Enabled
+  name: master
+  platform:
+    aws:
+      iamRole: ExampleRole
+----
+. Save the file and reference it when installing the {product-title} cluster.
```

Comment:
Suggestion: Since you reference the install method that makes use of `install-config.yaml`, I would like to see an Additional resources section xref in the parent assembly that links to a complete procedure that shows how this works on AWS.

Reply:
Agreed. Thank you.